src/title.tex

```diff
@@ -0,0 +1,52 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%% Title page
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+\begin{titlepage}
+    % \vspace*{5cm}
+    \makeatletter
+    \begin{center}
+        \begin{LARGE}
+            Eberhard Karls Universität Tübingen
+        \end{LARGE}
+        \begin{large}
+            Mathematisch-Naturwissenschaftliche Fakultät \\
+            Wilhelm-Schickard-Institut für Informatik
+        \end{large}
+        \vfill
+        \begin{huge} 
+            Master's Thesis
+        \end{huge}
+        \\\vspace{2cm}
+        \begin{Large} \textbf{\@title} \end{Large}
+        \\\vspace{1.5cm}
+        \begin{large} \@author \end{large}
+        \\\vspace{0.5cm}
+        {\small \handindate}
+        \vfill
+        {\small \textbf{Reviewers}}
+        \\
+        %\vspace{0.25cm}
+        \large{\firstReviewer}\\
+        \footnotesize{\firstReviewerAffiliation}
+        \\\vspace{0.5cm}
+        \large{\secondReviewer}\\
+        \footnotesize{\secondReviewerAffiliation}
+    \end{center}
+\end{titlepage}
```

Comment:
Last time, I used something like the following where the latter part should work correctly for 1-2 reviewers and different titles
```
\begin{titlepage}
 \begin{center}
  {\LARGE Eberhard Karls Universität Tübingen}\\
  {\large Mathematisch-Naturwissenschaftliche Fakultät \\
Wilhelm-Schickard-Institut für Informatik\\[4cm]}
  {\huge \ThesisTypeTitle\\[2cm]}
  {\Large\bf  \ThesisTitle\\[1.5cm]}
 {\large \ThesisAuthor}\\[0.5cm]
\ThesisSubmissionDate\\[4cm]
{\small\bf Reviewers}\\[1.0cm] % Note: is 0.5cm in the "original" template!
  \ifx\SecondReviewerName\empty
    \begin{tabular}{@{}c@{}@{}}
      {\large \FirstReviewerName{}} \\
      {\footnotesize \FirstReviewerDepartment{}} \\
      {\footnotesize \FirstReviewerUniversity{}}
    \end{tabular}
  \else
    \begin{tabular}{@{}P{6.5cm}@{\hspace{0.5cm}}P{6.5cm}@{}}
      {\large \FirstReviewerName{}} & {\large \SecondReviewerName{}} \\
      {\footnotesize \FirstReviewerDepartment{}} & {\footnotesize \SecondReviewerDepartment{}} \\
      {\footnotesize \FirstReviewerUniversity{}} & {\footnotesize \SecondReviewerUniversity{}}
    \end{tabular}
  \fi
 \end{center}
\end{titlepage}

\vspace*{\fill}
\begin{minipage}{11.2cm}
\textbf{\ThesisAuthor:}\\
\emph{\ThesisTitle}\\
\ThesisTypeTitle\\
Eberhard Karls Universität Tübingen\\
Thesis period: \ThesisPeriod
\end{minipage}
\newpage
```

where the additional defns were:
```

\def\ThesisTitle{\xxx{Thesis title}}
\def\ThesisAuthor{\xxx{YourName YourSurname}}
\def\ThesisTitlePDF{Thesis title}
\def\ThesisAuthorPDF{YourName YourSurname}
\def\ThesisSubmissionDate{\xxx{Datum}}
\def\ThesisPeriod{\xxx{von-bis}}

\def\ThesisAbstract{%
    \xxx{The abstract is a short summary of the work to be presented in the thesis}
}
\def\ThesisSummaryDE{%
    \xxx{Bei einer englischen Masterarbeit muss zus\"atzlich eine deutsche Zusammenfassung verfasst werden.}
}

\def\FirstReviewerName{\xxx{Name Erstgutachter}}
\def\FirstReviewerDepartment{\xxx{Wilhelm-Schickard-Institut für Informatik}}
\def\FirstReviewerUniversity{\xxx{Universität Tübingen}}

% Set name to empty => no second reviewer (for a bachelor thesis)
% \def\SecondReviewerName{}
\def\SecondReviewerName{\xxx{Name Zweitgutachter}}
\def\SecondReviewerDepartment{\xxx{Wilhelm-Schickard-Institut für Informatik}}
\def\SecondReviewerUniversity{\xxx{Universität Tübingen}}

\def\ThesisTypeTitle{Bachelor Thesis}
% \def\ThesisTypeTitle{Master Thesis}
\def\ThesisTypeGerman{Bachelorarbeit}
% \def\ThesisTypeGerman{Masterarbeit}
```

Reply:
The first case of the if expression never seems to trigger. Are there some packages I need to include for this to work?